### PR TITLE
implementation changes

### PR DESCRIPTION
**Pull request by Mostafa-Samir**
#### Changes Include:
1. Removing the call to `dynamic_rnn` outside the loop, hence losing the need for `scope.reuse_variables` and possibly making the model a bit faster.
2. Removing the `fully_connected` layer before batch the normalization because it doesn't seem necessary at all, and used the `update_ops` as it's reported to be faster than updating at place.
3. Initialize bias of first MLP to 0.

There might be some bugs while training as I didn't start training at all.

### CODE REVIEW

File: drnn.py [hunk 1]
```diff
@@ -124,59 +124,66 @@ def logits(self):
...
+            '''
+            Creating windowed input
+            In this step we take the input tensor of shape (batch_size, sequence_length, embedding_size)
+            and generate a tensor of shape (batch_size, sequence_length, window_size, embedding_size)
+            The new tensor will hold all k input for each time step in the sequence
+            '''
+            windowed_input = []
+            for t in range(self.sequence_length):
+                windowed_input.append(
+                    input[:, tf.newaxis, t:t + self.num_k, :]  # shape: (batch_size, 1, window_size, embedding_size)
```
Comment (ibrahimsharaf) [inline drnn.py]: What does tf.newaxis do?
